src/cpuid-templates/templates/t2s.json

```diff
@@ -0,0 +1,353 @@
+[
+   {
+      "function":0,
+      "index":0,
+      "flags":0,
+      "eax":13,
+      "ebx":1970169159,
+      "ecx":1818588270,
+      "edx":1231384169
```

Comment:
just out of curiosity, where are these values calculated from? I guess Intel's dev manual? Is there anything we can do to spot errors in these?

Reply:
These values come from saving the output of CPUID when called from within a guest microVM started with the `t2s` template (with the main Firecracker branch).

The latest Intel specification for CPUID is https://cdrdv2-public.intel.com/671368/architecture-instruction-set-extensions-programming-reference.pdf

I am not sure how we could spot errors here.